extensions/data-workspace/src/dialogs/openExistingDialog.ts

```diff
@@ -35,15 +35,15 @@ export class OpenExistingDialog extends DialogBase {
 
 	async validate(): Promise<boolean> {
 		try {
-			if (await this.workspaceService.validateWorkspace() === false) {
-				return false;
-			}
-
 			if (this.localRadioButton?.checked) {
 				await this.validateFile(this.filePathTextBox!.value!, constants.Project.toLowerCase());
 			} else {
 				await this.validateClonePath(<string>this.localClonePathTextBox!.value);
 			}
+
+			if (await this.workspaceService.validateWorkspace() === false) {
```

Comment:
I'm guessing it works fine, but it's a bit odd to have the validates above throwing an error instead of returning false.